database/migrations/2024_09_22_161700_add_class_standings.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Models\ClassStanding;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
```

Comment:
For the sake of the IDs also being in order, please reorder the inserts by ranking order